templates/iot.html

```diff
@@ -1,332 +1,416 @@
-  </div>
-  <div class="p-strip is-shallow">
-    <div class="row">
-      <div class="u-equal-height">
-        <div class="col-6 p-card">
-          <h3 class="p-card__title">自分で考えるロボット</h3>
-          <p class="p-card__content">ROS（Robotic Operating System）はUbuntu上で開発されています。コンピュータビジョン、AIライブラリ、そして幅広いセンサーのサポートにより、ドローンから自動運転車まで、自律動作マシン向けとして最も人気のある開発環境です。</p>
+    <section class="p-section--hero">
+        <div class="row--50-50">
+            <div class="col">
+                <h1>IoT/組み込み向けUbuntu</h1>
+            </div>
+            <div class="col">
+                <div class="p-section--shallow">
+                    <p>スマートホームからドローン、ロボット、産業システムまで、Ubuntuなら堅牢なセキュリティ、アプリストア、信頼性の高いアップデートをご利用いただけます</p>
+                </div>
+                <hr class="is-muted" />
+                <a href="/contact-us?product=iot" class="p-button--positive">お問い合わせ</a>
+                <a href="/engage/embedded-linux-yocto-ubuntu-core-whitepaper-jp"
+                   class="p-button">向け比較ガイド</a>
+            </div>
+        </div>
+    </section>
+    <section class="p-section">
+        <div class="row--50-50">
+            <hr />
+            <div class="col">
+                <h2>IoTにCanonicalを選ぶ理由</h2>
+            </div>
+            <div class="col">
+                <ul class="p-list--divided">
+                    <li class="p-list__item has-bullet">標準で最長10年間セキュリティメンテナンスをご提供</li>
+                    <li class="p-list__item has-bullet">リアルタイムカーネル及び低遅延カーネルのご提供</li>
+                    <li class="p-list__item has-bullet">グローバルレベルの拡張性に対応した堅牢なOTA更新インフラストラクチャ</li>
+                    <li class="p-list__item has-bullet">エッジ向けに最適化されたカスタム商用イメージ</li>
+                    <li class="p-list__item has-bullet">セキュアブートとフルディスク暗号化</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="p-section">
+        <div class="row--50-50 p-section--shallow">
+            <hr />
+            <div class="col">
+                <h2>開発から運用まで</h2>
+            </div>
+            <div class="col">
+                <div class="p-image-container p-section--shallow">
+                    <img class="p-image-container__image"
+                         src="https://assets.ubuntu.com/v1/85ebe155-IoT-Gateways-Kubernetes2.png" />
+                </div>
+                <p>
+                    開発から運用まで同一のカーネル、ライブラリ、システムソフトウェアを使いスムーズに本番環境へ移行可能です。アプリケーション開発はUbuntu DesktopまたはUbuntu
+                    Serverで行いアプリケーションとターゲットハードウェアに適した改変不能なUbuntu Coreイメージを構築します。
+                </p>
+            </div>
         </div>
-        <div class="col-6 p-card">
-          <h3 class="p-card__title">業界最高レベルのセキュリティ</h3>
-          <P class="p-card__content">Ubuntu Coreは、ロボットを動かすソフトウェアの整合性を確保するため、アプリケーション制限、ファイルアクセスの制限、読み取り専用アプリケーションを提供します。その結果、基板上のドライバーからロボットの動作を定義するアプリまで、設計上のセキュリティが確保されます。</P>
+        <div class="row">
+            <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+                <hr class="is-muted" />
```

Comment:
```suggestion
                <hr class="p-rule---muted" />
```
I've gotten this feedback recently so just passing it along, we're supposed to be moving away from the `is-muted` class and using `p-rule--muted` instead. Of course not for hrs that are just above buttons though (as in the hero) since this will remove the margin between the button and separator